pkgs/development/libraries/jcal/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, which, libtool, getopt, readline}:
+
+stdenv.mkDerivation rec {
+  name = "jcal";
+  version = "v0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "fzerorubigd";
+    repo = "jcal";
+    rev = "e4678bebb7efb7faa4435be3e87fc64da53e942d";
+    sha256 = "1b2nm499wcbpp2xim30b6sazkcg901cfr75qigclschj5rns6s9w";
+  };
+  doCheck = true;
+
+ # url = "https://github.com/fzerorubigd/jcal/";
+ # src = fetchFromGitHub {
+ #   owner = "fzerorubigd";
+ #   repo = "jcal";
+ #   rev="v0.4.1";
+ #   sha256 = "04jc24x9hmgmf6h3fczvs64bmj5kmvf934q0p0jmn2nsbrhdqpmm";
+ # };
+
+  buildInputs = [ libtool getopt readline autoconf automake ];
+  nativeBuildInputs = [ which ];
+
+  preConfigure = ''
+    cd sources/
+    ./autogen.sh
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Jalali Calendar Library";
+    homepage =  http://nongnu.org/jcal/;
+    license = licenses.gpl3;
+    maintainers = [ maintainers.linarcx ];
+    platforms = platforms.all;
+  };
+}
+
+#---=[ Garbage ]---#
```

Comment:
What's this comment?

Reply:
All comments can be removed now. i forgot to remove them.